src/MPAGSCipher/TransformChar.hpp

```diff
@@ -3,7 +3,6 @@
 
 #include <string>
 
-//! Transliterate char to string
-std::string transformChar(const char inputChar);
+std::string transformChar(const char in);
```

Comment:
While the code can be considered self-documenting, please keep the doxygen-style comments.